### ISSUE TITLE
Connection is Not Secure

### ISSUE THREAD

**Issue by Hasonk** [inuyaksa/jquery.nicescroll#707]
Work at good but "Connection is Not Secure" show on the URL Address of web browser  when insert the class name of boxscroll.
![2017-09-24_2-02-20](https://user-images.githubusercontent.com/16137250/30776207-a8cd1c94-a0cc-11e7-9281-fd4279829300.jpg)

**Comment by Prolitee2407** [comment]
You have to check if somewhere in you're source there is a link with http:// like this example : 
<script type="text/javascript" src="**http**://maps.google.com/maps/api/js?sensor=false"></script> then change it to <script type="text/javascript" src="**https**://maps.google.com/maps/api/js?sensor=false"></script>

**Comment by Hasonk** [comment]
Yes, I already checked and changed all links from 'http' to https.
I'm sure the problem is because of this plugin.
From the below picture, the connection is completely secure when I removed the class name of boxscroll.
![2017-09-27_6-09-57](https://user-images.githubusercontent.com/16137250/30888466-e5c95a10-a34a-11e7-88ad-b1d3e36a65aa.jpg)

**Comment by inuyaksa** [comment]
I check now, there is some reference of insecure url in your site not related to plugin.
Use Chrome or Firefox, open dev tool (F12) check network, in the list you find ad insecure connection to 
http://mesukgrandhotel.com/
